### PR TITLE
add auto start from snapshot

### PR DESCRIPTION
**Pull request by FineKe**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okx/okbchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okx/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: app/start_from_snapshot.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+	_, err := os.Stat(outputPath)
+	if err != nil {
+		os.MkdirAll(outputPath, 0755)
+	}
+
+	fileName := url[strings.LastIndex(url, "/")+1:]
+	targetFile := filepath.Join(outputPath, fileName)
+
+	// check file exists
+	if _, err := os.Open(targetFile); err == nil {
```
Comment (giskook) [inline app/start_from_snapshot.go:80]: Is there a better way to check file exists?

File: app/start_from_snapshot.go [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+		os.Remove(targetFile)
+	}
+
+	//maxSpeed := fmt.Sprintf("%d", viper.GetInt(server.FlagMaxDownloadSnapshotSpeed)*1024*1024)
+	var stdoutProcessStatus bytes.Buffer
+
+	axel := exec.Command("axel", "-n", fmt.Sprintf("%d", runtime.NumCPU()), "-o", targetFile, url)
```
Comment (giskook) [inline app/start_from_snapshot.go:87]: `axel` is our provided tools for fast download? if not installed, what will happen?

Comment (FineKe) [reply app/start_from_snapshot.go:87]: it got installed in the base image

File: app/start_from_snapshot.go (hunk 1 above)
Comment (FineKe) [reply app/start_from_snapshot.go]: yeah, useing os.Stat function

**Review by giskook — APPROVED** [review]

File: app/start_from_snapshot.go [hunk 3]
```diff
@@ -0,0 +1,199 @@
+package app
```
Comment (ras9481) [inline app/start_from_snapshot.go:1]:
```suggestion
package app
```

Comment (ras9481) [reply app/start_from_snapshot.go:1]:
```suggestion
package app
```

**Pull request merged**